### PR TITLE
[CIAC-6775] [PAN-OS] Add audit-comment argument

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-6775](https://jira-dc.paloaltonetworks.com/browse/CIAC-6775)

## Description
1. Added a new argument `audit-comment` to the following commands:
    - `pan-os-create-rule`
    - `pan-os-create-pbf-rule`
    - `pan-os-create-nat-rule`
    - `pan-os-edit-pbf-rule`
    - `pan-os-edit-nat-rule`
3. Added the new command `pan-os-get-audit-comment`.
4. Added `ip-wildcard` in `pan-os-list-addresses` result.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -5148,6 +5148,8 @@ script:
       - bottom
     - description: The destination rule relative to the rule that you are moving. This field is only relevant if you specify "before" or "after" in the "where" argument.
       name: dst
+    - description: A audit comment for the rule.
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5151]:
```suggestion
    - description: An audit comment for the rule.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 2]
```diff
@@ -8735,6 +8739,7 @@ script:
       - destination_translation_dynamic_ip
       - destination_translation_dynamic_distribution_method
       - disabled
+      - audit-comment
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:8742]: let's add to the RN as well

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -13660,6 +13704,49 @@ def pan_os_delete_tag_command(args: dict) -> CommandResults:
...
+    return CommandResults(
+        raw_response=raw_response,
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f'Audit Comment for Rule: {rule_name}',
+            outputs,
+            headerTransform=string_to_table_header,
+        ),
+        outputs_prefix='Panorama.AuditComment',
+        outputs_key_field='rule_name'
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:13746]: is a rule name unique identifier or need the rule type here as well?

**Review by dantavori — COMMENTED** [review]
nice, let's ask for a doc review
will re-review after the demo

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 3 above)
Comment (yaakovpraisler) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:13746]: Probably need also by type. Added.

File: Packs/PAN-OS/ReleaseNotes/2_1_23.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added a new command ***pan-os-get-audit-comment***.
+- Added the new argument *audit-comment* to the following commands:
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/2_1_23.md:7]:
```suggestion
- Added the *audit-comment* argument to the following commands:
```

**Comment by ShirleyDenkberg** [comment]
@dantavori @jlevypaloalto Doc review completed.

**Pull request merged**